README.md

```diff
@@ -202,4 +206,89 @@ exit 1
 
 This example asset:
 - Responds to `_assetInfo` with the required JSON and type.
-- Accepts only a single argument (the JSON object) and outputs a valid JSON object when invoked.
\ No newline at end of file
+- Accepts only a single argument (the JSON object) and outputs a valid JSON object when invoked.
+
+---
+
+## Conjure Backcompat Asset
+
+The `backcompat` asset type enables backward compatibility checking of Conjure APIs.
+
+### Requirements
+
+**Type Declaration:**
+- The asset must respond to the `_assetInfo` probe with:
+  ```json
+  { "type": "backcompat" }
+  ```
+
+**Invocation:**
+- The asset is invoked with a JSON object that uses a discriminated union pattern based on operation type.
+
+**Check Backcompat Mode (`checkBackCompat`):**
+```json
+{
+  "type": "checkBackCompat",
+  "checkBackCompat": {
+    "currentIR": "/tmp/current-ir.json",
+    "project": "my-api",
+    "groupId": "com.example",
+    "projectConfig": {
+      "output-dir": "./pkg",
+      "server": true,
+      "ir-locator": { "type": "yaml", "locator": "./conjure" }
+    },
+    "godelProjectDir": "/path/to/project"
+  }
+}
+```
+
+**Accept Backcompat Breaks Mode (`acceptBackCompatBreaks`):**
+```json
+{
+  "type": "acceptBackCompatBreaks",
+  "acceptBackCompatBreaks": {
+    "currentIR": "/tmp/current-ir.json",
+    "project": "my-api",
+    "groupId": "com.example",
+    "projectConfig": {
+      "output-dir": "./pkg",
+      "server": true,
+      "ir-locator": { "type": "yaml", "locator": "./conjure" }
+    },
+    "godelProjectDir": "/path/to/project"
+  }
+}
+```
+
+### Usage
+
+**Running Backcompat Checks:**
+```bash
+# Check all projects
+./godelw conjure-backcompat
+
+# Check a specific project
+./godelw conjure-backcompat --project=my-api
```

Comment:
I agree that this is behavior that we want to support, but it looks like none of the other commands in the plugin currently support specifying projects.

For this initial PR, let's match the other behavior of just doing it for all projects.

In a follow-up PR, we can add per-project support for all commands.

Also, for most user-facing godel commands, targets tend to be arguments rather than flags, so would likely prefer to go with that approach for consistency.